tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -219,3 +219,150 @@ def test_list_all_columns_with_fake_experiment_id(init_store):
                                tags=[])
     actual_columns = init_store.list_all_columns("fake_id", ViewType.ALL)
     assert expected_columns.__dict__ == actual_columns.__dict__
+
+
+@pytest.mark.parametrize("expected_runs_ids,test_filter_string",
+                         [(["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a"],
+                           'params.param0="valeur4"'),
+                          ([], 'params.param0="test3"'),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a",
+                            "d57a45f3763e4827b7c03f03d60dbbe1"], 'params."param0" LIKE \'%va%\''),
+                          ([], 'params."param0" LIKE \'%tes%\''),
+                          (["1e5200ae248b476cb0e60286e3f061a4"], 'tags.tag2="val2"'),
+                          ([], 'tags.tag0="test3"'),
+                          (["d57a45f3763e4827b7c03f03d60dbbe1"], 'tags."tag0" LIKE \'%Va%\''),
+                          ([], 'tags."tag0" LIKE \'%tes%\''),
+                          (["d57a45f3763e4827b7c03f03d60dbbe1"], 'tags."tag0" LIKE \'%Va%\''),
+                          (["d57a45f3763e4827b7c03f03d60dbbe1"], 'metrics.metric0=0'),
+                          (["1e5200ae248b476cb0e60286e3f061a4"], 'metrics.metric0>0'),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "d57a45f3763e4827b7c03f03d60dbbe1"],
+                           'metrics.metric0>=0'),
+                          (["4baa8e505cdb49109b6819a497f1a58a"], 'metrics.metric0<0'),
+                          (["4baa8e505cdb49109b6819a497f1a58a", "d57a45f3763e4827b7c03f03d60dbbe1"],
+                           'metrics.metric0<=0'),
+                          (["1e5200ae248b476cb0e60286e3f061a4", "4baa8e505cdb49109b6819a497f1a58a",
+                            "d57a45f3763e4827b7c03f03d60dbbe1"], '')])
+@pytest.mark.usefixtures('init_store')
+def test__search_runs_simple_filter(expected_runs_ids, test_filter_string, init_store):
+    actual_runs, next_page_token = init_store._search_runs(experiment_ids=["hjb553MBNoOYfhXjp3Tn"],
+                                                           filter_string=test_filter_string,
+                                                           run_view_type=ViewType.ACTIVE_ONLY)
+    for i, run in enumerate(actual_runs):
+        assert run._info.run_id == expected_runs_ids[i]
+
```

Comment:
You can also prepare the ILIKE tests with xfail annotation (see a comment bellow with the doc). It will help the dev who will fix it!